docs/attr.md

```diff
@@ -40,7 +40,9 @@ canViewCallbacks.attr("show-when", function(el, attrData){
 that matches attribute names. Examples: `"my-fill"` or `/my-\w/`.  
 
 @param {function(HTMLElement,can-view-callbacks.attrData)} attrHandler(el, attrData)
-A function that adds custom behavior to `el`.  
+A function that adds custom behavior to `el`. Note that `el` may be a
```

Comment:
I think it's good enough to just say that `el` might not be in the DOM when the callback is called. I think the extra information about it possibly being a DocumentFragment might just confuse people.